test/base/LendingPoolWithdrawTest.js

```diff
@@ -33,14 +35,15 @@ contract('LendingPoolWithdrawTest', function (accounts) {
     });
 
     withData({
-        _1_basic: [accounts[0], true, 10, false, undefined, false],
-        _2_transferFail: [accounts[1], false, 50, false, 'LENDING_TRANSFER_FAILED', true],
-        _3_compoundFail: [accounts[1], true, 50, true, 'COMPOUND_WITHDRAWAL_ERROR', true],
+        _1_basic: [accounts[0], true, 10, false, 1000, undefined, false],
+        _2_transferFail: [accounts[1], false, 50, false, 1000, 'LENDING_TRANSFER_FAILED', true],
+        _3_compoundFail: [accounts[1], true, 50, true, 1000, 'COMPOUND_WITHDRAWAL_ERROR', true],
```

Comment:
Is it possible to add a new scenario like _1_ but the allowance 0 (or less than amountToWithdraw) to get a balance error?

Reply:
Added